### PR TITLE
VACMS-7721: display published status on non-clinical service cards

### LINKED ISSUE #7721: Publishing status of nonclinical service shows on location card
## User Story 

As an editor looking at a top task page, I need an easy way to see the publishing state of the associated facility nonclinical service locations (i.e. the location cards), so that I know whether I need to take action. 

Possible solution: a colored bar across the top of the location card that indicates the location's publishing status

## Design principles

Veteran-centered
- [ ] `Single source of truth`: Increase reliability and consistency of content on VA.gov by providing a single source of truth.
- [ ] `Accessible, plain language`: Provide guardrails and guidelines to ensure content quality.
- [ ] `Purposely structured content`: Ensure Content API can deliver content whose meaning matches its structure.
- [ ] `Content lifecycle governance`: Produce tools, processes and policies to maintain content quality throughout its lifecycle.

Editor-centered
- [ ] `Purpose-driven`: Create an opportunity to involve the editor community in VA’s mission and content strategy goals.
- [X] `Efficient`: Remove distractions and create clear, straightforward paths to get the job done.
- [ ] `Approachable`: Offer friendly guidance over authoritative instruction.
- [ ] `Consistent`: Reduce user’s mental load by allowing them to fall back on pattern recognition to complete tasks.
- [ ] `Empowering`: Provide clear information to help editors make decisions about their work.

### CMS Team

Please leave only the team that will do this work selected.
- [ ] `Platform CMS Team`
- [x] `Sitewide CMS Team`
  - [ ] `⭐️ Content ops`
  - [ ] `⭐️ CMS experience`
  - [ ] `⭐️ Offices`
  - [x] `⭐️ Product support`
  - [ ] `⭐️ User support`

### PR DESCRIPTION
**Pull request by JayDarnell**
## Description

Closes #7721

## Testing done

Visual Testing

## Screenshots

VA.gov Claro Theme
Screen Shot 2022-02-08 at 10 07 00 AM

VA.gov Admin Theme
Screen Shot 2022-02-08 at 10 43 01 AM

## QA steps

As an admin user:
- [ ] Visit /admin/appearance and set the default and admin themes to VA.gov Claro
- [ ] Visit /admin/content and filter the content for section: VA Eastern Colorado health care
- [ ] Further filter the content for content type: VAMC System Billing and Insurance (Edit the existing node)
- [ ] Verify that the background color for the listed non-clinical service nodes match the actual published state (pink background for unpublished)
- [ ] On /admin/content change the content type to VAMC System Medical Records Office (Edit the existing node)
- [ ] Verify that the background color for the listed non-clinical service nodes match the actual published state (pink background for unpublished)
- [ ] On /admin/content change the content type to VAMC System Register for Care (Edit the existing node)
- [ ] Verify that the background color for the listed non-clinical service nodes match the actual published state (pink background for unpublished)

### Select Team for PR review

- [ ] `Platform CMS Team`
- [ ] `Sitewide CMS Team`
  - [ ] `⭐️ Content ops`
  - [ ] `⭐️ CMS Experience`
  - [ ] `⭐️ Offices`
  - [x] `⭐️ Product Support`

### CODE REVIEW

**Comment by JayDarnell** [comment]
@swirtSJW this PR still has the issue we discussed with caching in the va.gov admin theme but maybe that is okay as we are switching to Claro?

File: docroot/themes/custom/vagovadmin/templates/views/views-view-grouping--non-clinical-services.html.twig [hunk 1]
```diff
@@ -14,9 +14,16 @@
  * @see template_preprocess_views_view_grouping()
  */
 #}
+{% if grouping_level == 0 %}
```
Comment (swirtSJW) [inline docroot/themes/custom/vagovadmin/templates/views/views-view-grouping--non-clinical-services.html.twig:17]: Still would like a comment here leaving a breadcrumb about why this was necessary.  It is not obvious to  me and definitely will not be obvious in a month from now.

File: docroot/themes/custom/vagovclaro/templates/views/views-view-grouping--non-clinical-services.html.twig [hunk 2]
```diff
@@ -14,9 +14,16 @@
  * @see template_preprocess_views_view_grouping()
  */
 #}
+{% if grouping_level == 0 %}
+<div class="view-grouping {{ content|first['#title']|render|striptags|trim|lower }}">
```
Comment (swirtSJW) [inline docroot/themes/custom/vagovclaro/templates/views/views-view-grouping--non-clinical-services.html.twig:18]: comment please

**Review by swirtSJW — CHANGES_REQUESTED** [review]
This all looks good.  Nice work.  Just would like comments in both templates leaving a bit of a clue about the why.

**Pull request merged**